build/index.js

```diff
@@ -25,7 +25,7 @@ var StringifyWithFloats = function StringifyWithFloats() {
       } else {
         value = val;
       }
-      var forceFloat = config[key] === 'float' && (value || value === 0) && typeof value === 'number';
+      var forceFloat = config[key] === 'float' && (value || value === 0) && typeof value === 'number' && !value.toString().includes('e');
```

Comment:
Should we also check for uppercase `E`? 
It also seems like `.toString` returns the numerical value and not the scientific notation
![screenshot 2019-01-14 12 53 14](https://user-images.githubusercontent.com/17786035/51137123-5dd54980-17fb-11e9-92a3-47ff92f96724.png)

Reply:
that's because it's not a very large/small number

```js
> 5e-7
5e-7
```

but yes, i'll check for uppercase also 👍